advent_calendar.md

```diff
@@ -229,10 +228,10 @@ in the file:
 {"index":{"_id":"job_model_state_1484006460"}}
```

Comment:
I've removed all this xml entirely, it doesn't exactly add anything to the story - plus it put me well over the character limit!